ccmlib/cluster.py

```diff
@@ -616,6 +616,13 @@ def drain(self):
     def repair(self):
         self.nodetool("repair")
 
+        stdout, _ = self.nodetool('help')
```

Comment:
also this whole logic should go into `ScyllaCluster.repair()`, it's doesn't have any relevancy for this file...